docker-compose-prod.yml

```diff
@@ -0,0 +1,63 @@
+# Version 3, most current, and recommended compose file version
+# https://docs.docker.com/compose/compose-file/compose-versioning/
+version: '3'
+
+services:
+  # First service
+  telescope:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file: .env
+    environment:
+      # a place-holder for the variable passed by CLI with a default value
+      # the passed or default value is a command to run `telescope`
+      - script=${script:-start}
+      # override the .env variable for `REDIS_URL`
+      - REDIS_URL=redis://redis:${REDIS_PORT}
+    depends_on:
+      - redis
+    ports:
+      - '$PORT:$PORT'
+    restart: always
```

Comment:
I think this has been replaced with `restart_policy`, see https://docs.docker.com/compose/compose-file/#restart_policy

Reply:
I tried to add `restart_policy`, and it needs a `deploy` key, which is not supported by compose. I might need to revert to `restart:always` (if I don't use `deploy`, compose complains).
![deploy](https://user-images.githubusercontent.com/23108901/73807970-42d5ce80-479c-11ea-970e-400c21698299.png)

Reply:
Noted.  Thanks for testing.